roles/rabbitmq/server/tasks/main.yml

```diff
@@ -0,0 +1,44 @@
+---
+- name: install all rpms
+  yum:
+   name=rabbitmq-server
+   state=present
+  tags: rabbitmq
+
+- name: start the service
+  service:
+   name=rabbitmq-server
+   state=started
+   enabled=yes
+  tags: rabbitmq
+
+- rabbitmq_plugin:
+   names=rabbitmq_management
+   state=enabled
+  tags: rabbitmq
```

Comment:
I would remove these tags.  Unless we have a comprehensive structure for tag use in this repository, they just end up being confusing.  Ideally, they should only be set at the top level plays on the `role:` sections instead of per-role.